### ISSUE TITLE
Make view link trigger a new window

### ISSUE THREAD

**Issue by eladnova** [ryancramerdesign/ProcessWire#1073]
Could we consider making the view links trigger a new tab/window?
I've always found it a bit odd that it replaces the existing window.

The same behaviour should be applied to the home button in Reno Theme.

There seems to be some community support for this (https://processwire.com/talk/topic/9278-processwire-feedback-from-training-session)

**Comment by mike-anthony** [comment]
+1 :+1:

**Comment by ryancramerdesign** [comment]
ProcessWire is intended for single-window workflow, though it's fine to open a new window when that's your intention. To open a new tab/window, do cmd-click on mac, or ctrl-click in Windows when clicking a view link. Or right click and choose "open in new window/tab". Forcing a new window/tab would prevent the user from being able to make the decision, a loss of functionality. When opening multiple windows, be aware that it causes issues with the user clicking the front-end edit button in one window while still having an editing session in another, and potentially overwriting changes when they save. It can also interfere with the CSRF protection in some cases. If using the SystemNotifications module, it puts unnecessary load on the server and your connection to have multiple open to the same PW admin (not an issue on front-end though). If you are worried about new users thinking something is saved before they saved it, Soma's PageEditSoftLock module is a good way to double check with them if they want to save first.

**Comment by wanze** [comment]
Maybe we could add this behaviour as a config option to ProcessPageEdit? :)

**Comment by eladnova** [comment]
Yep - I get around it by right-clicking and doing the new tab thing. Not a problem.
Actually, I usually have a separate window open for fields, templates, pages and then the www site.

It was more a request from watching clients interact with the admin. Appreciate the tech reasons though.

**Comment by ryancramerdesign** [comment]
> Maybe we could add this behaviour as a config option to ProcessPageEdit? :)

Yes, I think the only way I can accommodate this request is to make it a configuration option. Though for people using it, it seems like we would then need more things built-in to prevent the possibility of multiple windows editing the same page. At least, that's the trouble I always get myself into when cmd-clicking the view link (which inevitably leads me back into the editor).
